### ISSUE TITLE
Error using acts_as_heir_of when use underscore '_' in a model filename

### ISSUE THREAD

**Issue by henriquesobral** [dipth/Heritage#8]
Hi,

When i try use your software to inherit my <strong>im_image</strong> table from <strong>im_file</strong> model, i received the following error

<strong>Expected /Users/hsobral/sistemas/imager/app/models/im_file.rb to define Im_file</strong>

I founded this two lines using capitalize. I think the right would be to use the camelize

``` ruby
self._predecessor_klass = Object.const_get(predecessor_symbol.to_s.capitalize)
...
has_one :predecessor, :as => :heir, :class_name => predecessor_symbol.to_s.camelize, :autosave => true, :dependent => :destroy
```

Thanks.
Henrique

**Comment by ekampp** [comment]
You might be very right, I will look into it over the weekend.

**Comment by dipth** [comment]
@henriquesobral could you please check if this has been fixed in version 0.3.2
